charts/vald/values.yaml

```diff
@@ -1058,7 +1058,7 @@ gateway:
       pathType: ImplementationSpecific
       # @schema {"name": "gateway.lb.ingress.enabled", "type": "boolean"}
       # gateway.lb.ingress.enabled -- gateway ingress enabled
-      enabled: false
+      enabled: true
```

Comment:
It would be better to set `false`. 🙏  If we do not have a controller for ingress, we cannot create an ingress resource and this deployment will fail.
Since I think `values.yaml` should work under any environment, it makes sense to let the user set it to enable if we want to use ingress.